### PR TITLE
Resizable: Modified to allow jquery objects as handles

### PR DESCRIPTION
**Pull request by tjvantoll**
Custom handlers did not work as jquery objects (outside the resizable element)

Fixes #9658

### CODE REVIEW

File: ui/resizable.js [hunk 1]
```diff
@@ -195,7 +196,11 @@ $.widget("ui.resizable", $.ui.mouse, {
 
 		this._renderAxis = function(target) {
 
-			var i, axis, padPos, padWrapper;
+			var i, axis, padPos, padWrapper, mouseDownHandlers = {
```
Comment (scottgonzalez) [inline ui/resizable.js]: missing newline

File: ui/resizable.js [hunk 2]
```diff
@@ -195,7 +196,11 @@ $.widget("ui.resizable", $.ui.mouse, {
...
+				mousedown: function( event ) {
```
Comment (scottgonzalez) [inline ui/resizable.js]: Can't this simply be `mousedown: "_mouseDown"`?

File: ui/resizable.js [hunk 3]
```diff
@@ -203,6 +208,9 @@ $.widget("ui.resizable", $.ui.mouse, {
 
 				if (this.handles[i].constructor === String) {
 					this.handles[i] = this.element.children( this.handles[ i ] ).first().show();
+				} else if ( this.handles[ i ].jquery || this.handles[i].nodeType ) {
```
Comment (scottgonzalez) [inline ui/resizable.js]: Missing spaces inside braces.

File: ui/resizable.js (hunk 2 above)
Comment (scottgonzalez) [inline ui/resizable.js]: And once this is simplified, we can just inline the object in the `_on()` call ;-)

File: ui/resizable.js [hunk 4]
```diff
@@ -219,11 +227,12 @@ $.widget("ui.resizable", $.ui.mouse, {
...
 
 					this._proportionallyResize();
-
 				}
 
+				this._handles = this._handles.add( this.handles[ i ] );
+
 				// TODO: What's that good for? There's not anything to be executed left
-				if (!$(this.handles[i]).length) {
+				if ( !$( this.handles[ i ] ).length ) {
```
Comment (scottgonzalez) [inline ui/resizable.js]: Seems like a good time to just drop this and get rid of that TODO :-)

File: ui/resizable.js [hunk 5]
```diff
@@ -232,8 +241,8 @@ $.widget("ui.resizable", $.ui.mouse, {
 		// TODO: make renderAxis a prototype function
 		this._renderAxis(this.element);
 
-		this._handles = $(".ui-resizable-handle", this.element)
-			.disableSelection();
+		this._handles = this._handles.add( $( ".ui-resizable-handle", this.element ) );
```
Comment (scottgonzalez) [inline ui/resizable.js]: `this.element.find()`; we shouldn't use jQuery objects as contexts.

**Comment by tjvantoll** [comment]
@patrixd Are you interested in continuing this and implementing @scottgonzalez's suggestions? Otherwise I can go through them and patch this up.

**Comment by tjvantoll** [comment]
@scottgonzalez I agreed with all of your suggestions and just pushed updates. Much cleaner.

**Comment by patrixd** [comment]
Thanks so much @tjvantoll for your support. I did not have time this week at work to push the updates.

**Comment by scottgonzalez** [comment]
Looks good.

**Pull request closed without merging**